scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,74 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(samples, q=(0.25, 0.75), axis=None, interpolation='lower'):
```

Comment:
Please use `flake8` to check for PEP8 issues. These will cause a CI failure:

![image](https://user-images.githubusercontent.com/6570539/106397793-11e6a480-63c4-11eb-9e79-ae72cb770c6b.png)

This and a few other lines are too long. You only need to fix the ones that correspond with your changes, though. (We weren't as careful in the past about this sort of thing.)